aws/data_source_aws_mq_broker.go

```diff
@@ -84,6 +88,14 @@ func dataSourceAwsMqBroker() *schema.Resource {
 					},
 				},
 			},
+			"enable_logging": {
+				Type:     schema.TypeBool,
+				Computed: true,
+			},
+			"enable_auditing": {
+				Type:     schema.TypeBool,
+				Computed: true,
+			},
```

Comment:
I would consider using a `TypeList` with `MaxItems: 1` to align the HCL a bit more closely with [the underlying AWS API](https://docs.aws.amazon.com/amazon-mq/latest/api-reference/rest-api-brokers.html#rest-api-brokers-attributes-logs-table). Something like this:

```hcl
logging {
  general = true
  audit   = true
}
```

Reply:
🤔 I dig it too. Will change in a bit.